models/public/hrnet-v2-c1-segmentation/model.yml

```diff
@@ -0,0 +1,136 @@
+  - name: mit_semseg/lib/nn/modules/replicate.py
+    size: 3226
+    sha256: 4498b873404fce05202d4f891262e249a95733816d3877e3aee3d2401138a6af
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/lib/nn/modules/replicate.py
+  - name: mit_semseg/models/__init__.py
+    size: 53
+    sha256: 9117afeac26d724bdba6f9f35d54b7ddab81e33847e717071c6ed789a458d84b
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/__init__.py
+  - name: mit_semseg/models/hrnet.py
+    size: 16811
+    sha256: f6e07d7448746b101a30f7b250896a9651b99fa6e02820afcfa69b93a8dd9a1d
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/hrnet.py
+  - name: mit_semseg/models/mobilenet.py
+    size: 4938
+    sha256: 2462300b9863d40da75d2cbcf785015e53930c3ce37219cb9d4c24b468b63a34
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/mobilenet.py
+  - name: mit_semseg/models/models.py
+    size: 21185
+    sha256: fea2849abca35a260a64e852397bcccad64e2fd495fad668ad1622b5080033eb
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/models.py
+  - name: mit_semseg/models/resnet.py
+    size: 6770
+    sha256: 685bb02446ce28eb1b110f3dfa9f6534a2ea34f50f09be0d7c165bd2c5c6c899
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/resnet.py
+  - name: mit_semseg/models/resnext.py
+    size: 5367
+    sha256: 271a95bdabb9e2c00ecc2e6765b7521f708cda5312b4d78bfb895374f3509044
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/resnext.py
+  - name: mit_semseg/models/utils.py
+    size: 577
+    sha256: 55a8f41269be28c17e456e5ae1f246bf71354765739dceca1e45becd58cba5e2
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/models/utils.py
+  - name: mit_semseg/__init__.py
+    size: 63
+    sha256: 7e58a465dbf83903f13d94ffce5b3353bcc8eb2154cd568bb405341f8a02ce5e
+    source: https://raw.githubusercontent.com/CSAILVision/semantic-segmentation-pytorch/9aff40de31ee4b21f18514d31e5d6e4ba056924d/mit_semseg/__init__.py
+  - name: ckpt/decoder_epoch_30.pth
+    size: 4780389
+    sha256: 6060764940b620804befaca4df916de019c82c4e96ace00916991e223af37d1e
+    source: http://sceneparsing.csail.mit.edu/model/pytorch/ade20k-hrnetv2-c1/decoder_epoch_30.pth
```

Comment:
oops! looks like downloading server is down now... downloading failed with time out error...

Reply:
Not a good sign... let's hope it was just an isolated incident.

Reply:
in the repo they have issues a year ago that time to time their server under maintenance.
Now http://sceneparsing.csail.mit.edu/ is unreachable, so I think it is also connected with some services on server side.

Reply:
it is alive! :)